packages/core/src/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from "./client";
 export * from "./client-types";
 export * from "./types";
+export * from "./transactions";
+export { circleAminoConverters, circleProtoRegistry } from "./codegen/circle/client"
```

Comment:
I'm not totally sure that exporting the circle proto types should fall under the responsibility of our library

Reply:
You're right. Removing those exports now